src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/cas1/Cas1AssessmentEmailService.kt

```diff
@@ -50,6 +50,25 @@ class Cas1AssessmentEmailService(
     }
   }
 
+  fun appealedAssessmentAllocated(allocatedUser: UserEntity, assessmentId: UUID, crn: String) {
+    allocatedUser.email?.let { email ->
```

Comment:
A wider point - we do this all over the place - is it expected that a user won't have an email? I wonder if we should be logging something if they don't (and come up with a clever way to do that everywhere). Maybe we should pass the user to the email service and let that do the ignore if null check and log if missing?

Reply:
I'm not 100% sure, I think we do this because `email` is nullable. Looking at preprod, we only have 4 users without email addresses.

Reply:
ok, i'll keep pondering on it :)